src/main/java/uk/ac/sanger/sccp/stan/service/operation/confirm/ConfirmSectionValidationServiceImp.java

```diff
@@ -91,6 +113,44 @@ public UCMap<Labware> validateLabware(Collection<String> problems, List<ConfirmS
         return lwMap;
     }
 
+    public UCMap<SlotRegion> validateSlotRegions(Collection<String> problems, List<ConfirmSectionLabware> csls) {
+        if (csls.stream().flatMap(csl -> csl.getConfirmSections().stream().map(ConfirmSection::getRegion))
+                .allMatch(BasicUtils::nullOrEmpty)) {
+            return new UCMap<>(0);
```

Comment:
Empty region validation check if there are  multiple sections is done on client side. Is it required in core side?

Reply:
I think you mean checking to make sure that regions are specified when there are multiple sections in the same slot. Maybe it would be a good idea to check that.

Reply:
Great 👍